docs/manual/docs/cmd/spo/site/site-remove.md

```diff
@@ -1,9 +1,17 @@
-# spo site classic remove
+# spo site remove
 
 Removes the specified site
 
 ## Usage
 
+```sh
+spo site remove [options]
+```
+
+## Alias
+
+The spo site classic remove command is deprecated.
```

Comment:
Deprecation warning doesn't belong in this section and should go either to remarks or be removed altogether